validator/validation/validation.py

```diff
@@ -333,24 +381,27 @@ def run_validation(validation_id):
 
         for async_result in async_results:
             try:
-                if not validation_aborted: ## only wait for this task if the validation hasn't been cancelled
+                if not validation_aborted:  # only wait for this task if the validation hasn't been cancelled
                     task_running = True
-                    while task_running: ## regularly check if the validation has been cancelled in this loop, otherwise we wouldn't notice
+                    while task_running:  # regularly check if the validation has been cancelled in this loop, otherwise we wouldn't notice
                         try:
-                            results = async_result.get(timeout=10) ## this throws TimeoutError after waiting 10 secs or TaskRevokedError if revoked before starting
-                            ## if we got here, the task is finished now
-                            task_running = False ## stop looping because task finished
-                            if celery_task_cancelled(async_result.id): ## we can still have a cancelled validation that took less than 10 secs
+                            results = async_result.get(
+                                timeout=10)  # this throws TimeoutError after waiting 10 secs or TaskRevokedError if revoked before starting
```

Comment:
strange formatting here, but I think this was some autoformatter? Maybe you can move the comment just above the line